docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -229,6 +229,30 @@ When a template's active version is updated:
 
 The system always maintains the desired number of prebuilt workspaces for the active template version.
 
+### Invalidating prebuilds
+
+When external dependencies change (such as AMI updates, base image changes, or configuration drift) without a template version update, you can invalidate presets to force their prebuilt workspaces to be recreated.
```

Comment:
nit: maybe removing the examples here and expanding on those on the next paragraph is clearer?

```suggestion
When external dependencies change without a template version update, you can invalidate presets to force their prebuilt workspaces to be recreated.
```